schema/PpaInformation.schema.json

```diff
@@ -21,8 +21,7 @@
       "type": "array",
       "description": "List of one or more CBSD Reference IDs in the cluster list of the PPA.",
       "items": {
-        "type": "string",
-		"pattern" : "cbsd/.+/.+"
+        "type": "string"
```

Comment:
Why change this?

Reply:
Actually I removed pattern check, according to sony review comment [5], because the dump can contain an empty field for this item, but I think removing it was not a good idea, because this check is valid even if the array is empty.